xbmc/pictures/Picture.cpp

```diff
@@ -161,24 +161,30 @@ bool CPicture::ResizeTexture(const std::string &image, uint8_t *pixels, uint32_t
   // create a buffer large enough for the resulting image
   GetScale(width, height, dest_width, dest_height);
 
-  uint8_t *buffer = new uint8_t[dest_width * dest_height * sizeof(uint32_t)];
+  // Let's align so that stride is always dividable by 16 and some 32 byte more on top
+  // See: https://github.com/FFmpeg/FFmpeg/blob/75638fe9402f70645bdde4d95672fa640a327300/libswscale/tests/swscale.c#L157
+  uint32_t dest_width_aligned = ((dest_width + 15) & ~0x0f);
+  uint32_t stride = dest_width_aligned * sizeof(uint32_t);
+
+  uint32_t* buffer = new uint32_t[dest_width_aligned * dest_height + 1];
```

Comment:
Now it would be factor sizeof(uint32_t) too large. Before you had: "number of chars", now it's "number of uint32_t

Reply:
Sorry! Disregard. All fine :-)

Reply:
Argh, missed the +1 - should be +4 (as we now have 4byte data type) and 4 * 4 = 16, while it was 16*1 before.